### ISSUE TITLE
Text file Encoding issue

### ISSUE THREAD

**Issue by philbibliography** [delax/bibliocraft-book-formatter#12]
Hello, not sure if you're still active, but I encountered a problem with the script.
I open the GUI app, type in everything I need to but when I try to start the converting process, this appears:

Exception in Tkinter callback
Traceback (most recent call last):
  File "C:\Python34\lib\tkinter\__init__.py", line 1487, in __call__
  File "formatter_gui.py", line 198, in write_some_books
  File "X:\Python34\lib\encodings\cp1250.py", line 23, in decode
UnicodeDecodeError: 'charmap' codec can't decode byte 0x98 in position 408: character maps to <undefined>

Can you tell me how to fix this please? I tried installing Python 3.4.0 and uninstalling the latest Pythin version and still nothing...

**Comment by delax** [comment]
It's been a bit, I'll fire it up and see if i get something similar.

**Comment by delax** [comment]
So, if you used the "**formatter_gui.exe**", then you do not need to install anything. It comes with "**python34.dll**" and other stuff. This lets it run all from the folder, without you installing python.

(That's what CXFreeze does, in case you wondered)


As for the error...

[test.txt](https://github.com/delax/bibliocraft-book-formatter/files/8896080/test.txt)
Hm it seems to work with this test.txt file.

Does this work for you @philbibliography ?
What were you trying to convert?

**Comment by delax** [comment]
Ok so that file and I got the same error, AND I got the error when running without the GUI.

Turns out it's something about the encoding of the file that makes my simple code sad?
But if it's saved as ANSI (which is more basic) it works.

TL;DR 

1. Open your file in Notepad
2. Save As
3. Change "Encoding:" to "ANSI"
4. Save it with new name
5. Use the New text file

**Comment by philbibliography** [comment]
Thank you so much, it works now!

**Comment by delax** [comment]
Excellent!
If I get a couple days, I'll look into the why and fix it.
Or at the very least, force convert to ANSI haha
